pkg/specprovider/specprovider.go

```diff
@@ -46,6 +48,17 @@ func (ppb *specProviderBuilder) New(client client.Client, builder configmanager.
 			return nil, err
 		}
 		return mgr, nil
+	case "LOCAL":
```

Comment:
Considering this, the end user for ARO/OCP would always have to specify the source as "LOCAL" in the configmap explicitly. What about considering LOCAL as the default source if no other source (Eg. OCM) is mentioned explicitly?

Reply:
We will have the configmap since there are some other places referencing the configs. 
I think it is not necessary to set a default value here.